config/endpoints/4_ministere_interieur_v4.yml

```
@@ -97,54 +97,53 @@
+  call_id: *call_id
+  provider_uids: *association_provider_uids
+  parameters: *association_parameters
+  keywords: *association_keywords
+  data:
+    description: |+
+      {:.fr-highlight.fr-highlight--caution}
+      ⚠️ **Le type et la quantité d'informations disponibles par association est variable** car de nombreuses ressources sont déclaratives ou déposées par les associations dans le cadre de démarches en ligne.
+
+      **Cette API délivre des informations publiques sur l'unité légale de l'association** :
+      - identité et statuts (forme juridique, dates clés, état de l'association, statut utilité publique, régime, adresse de gestion) ;
+      - tranche effectif de salariés ;
+      - activités principales ;
+      - agréments ;
+      - affiliations à des unions, fédérations ou réseaux, ainsi que des informations à leur sujet.
 
-#       **Cette API délivre des informations publiques sur l'unité légale de l'association** :
-#       - identité et statuts (forme juridique, dates clés, état de l'association, statut utilité publique, régime, adresse de gestion) ;
-#       - tranche effectif de salariés ;
-#       - activités principales ;
-#       - agréments ;
-#       - affiliations à des unions, fédérations ou réseaux, ainsi que des informations à leur sujet.
+      <br>
+      **Et sur les établissements de l'association** :
+      - état et adresse ;
+      - activités principales ;
+      - effectifs salariés, bénévoles et volontaires ;
+      - agréments.
 
-#       <br>
-#       **Et sur les établissements de l'association** : 
-#       - état et adresse ;
-#       - activités principales ;
-#       - effectifs salariés, bénévoles et volontaires ;
-#       - comptes de résultats et financement ;
-#       - agréments.
-    
-#       <br>
-#       **Ces données sont issues de trois bases de ressources différentes** :
-#       - le répertoire national des associations (RNA) ;
-#       - le répertoire Sirene ;
-#       - les données saisies par les associations dans [Le compte Asso](https://lecompteasso.associations.gouv.fr){:target="_blank"}, nommées documents administratifs complémentaires (DAC).
+      <br>
+      **Ces données sont issues de deux bases de ressources différentes** :
+      - le répertoire national des associations (RNA) ;
+      - le répertoire Sirene.
 
-#   opening: public
-#   format: *association_format
-#   availability: *association_availability
-#   use_cases:
-#     - Portail GRU - Préremplissage
-#   use_cases_optional:
-#     - Marchés publics
-#     - Aides & subventions publiques
-#     - Portail GRU - Instruction des démarches 
-#     - Détection de la fraude
-#   faq: *association_faq
+  opening: public
+  format: *association_format
+  availability: *association_availability
+  use_cases:
+    - Portail GRU - Préremplissage
+  use_cases_optional:
+    - Marchés publics
+    - Aides & subventions publiques
+    - Portail GRU - Instruction des démarches
+    - Détection de la fraude
```

En prévision de la PR qui arrive et qui ajoute le cas d'usage subventions associations :

```suggestion
    - Détection de la fraude
    - Subventions associations
```